tools/signtool/main.go

```diff
@@ -0,0 +1,215 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
funny enough this code was copied originally from https://github.com/10gen/notary-service/blob/master/client/signtool/signtool.go which does not have a license

Reply:
updated the comment to reflect the original place

Reply:
my point here even to the original had no license, meaning unlicensed, it's now covered by SSL since it was made available by compass, I'm not familiar enough with SSL to know, 1, do we need to include the license here instead of apache, usually that's a yes, 2 what the implications of including SSL in the code base are, technically we won't ship this code so it should not change the license of the CLI but IANAL